webapp/src/components/modals/run_playbook_modal.tsx

```diff
@@ -0,0 +1,226 @@
+import React, {ComponentProps, useState} from 'react';
+
+import {useIntl} from 'react-intl';
+import styled from 'styled-components';
+import {useSelector} from 'react-redux';
+import {getCurrentUser} from 'mattermost-redux/selectors/entities/users';
+
+import {Client4} from 'mattermost-redux/client';
+
+import Icon from '@mdi/react';
+import {mdiNotebookOutline, mdiAccountOutline} from '@mdi/js';
+
+import {GlobalState} from 'mattermost-redux/types/store';
+
+import {displayUsername, getFullName} from 'mattermost-redux/utils/user_utils';
+
+import {getTeammateNameDisplaySetting} from 'mattermost-redux/selectors/entities/preferences';
+
+import {usePlaybookName} from '../../hooks';
+import {BaseInput} from '../assets/inputs';
+import GenericModal, {InlineLabel, Description} from '../widgets/generic_modal';
+import {createPlaybookRun} from '../../client';
+import {navigateToUrl} from '../../browser_routing';
```

Comment:
nit: absolute paths (nothing against, just for consistency)
```suggestion
import {usePlaybookName} from 'src/hooks';
import {BaseInput} from 'src/components/assets/inputs';
import GenericModal, {InlineLabel, Description} from 'src/components/widgets/generic_modal';
import {createPlaybookRun} from 'src/client';
import {navigateToUrl} from 'src/browser_routing';
```

Reply:
Thank you @calebroseland for the detailed review. I will make the requested changes and update the PR in the next few days.